clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesServerGroup.groovy

```diff
@@ -77,6 +79,10 @@ class KubernetesServerGroup implements ServerGroup, Serializable {
   }
 
   Boolean isDisabled() {
+
```

Comment:
The exact approach isn't important, what I think is important is: 

1. We shouldn't be checking in JAR files.
2. The JARs should only be required when someone intends to develop against this K8S client.
3. I don't think we should host the JARs separately from maven.

One option avoiding all the build file configuration is that (and this isn't typically desired) that you work off of a separate fork until the JARs are published to maven central. This avoids all three points above, but will make the merge a little painful.